pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -38,7 +38,8 @@ var (
 	// MonOutTimeout is the duration to wait before removing/failover to a new mon pod
 	MonOutTimeout = 10 * time.Minute
 
-	timeZero = time.Duration(0)
+	retriesBeforeFailover = 1
```

Comment:
```suggestion
	retriesBeforeDrainFailover = 1
```

Reply:
did you mean `retriesBeforeMonFailover` ?

Reply:
Actually, since we retry once this could be a bool only, right?

Reply:
Yes, but `int` would help  if we want to increase the retry count in the future.

Reply:
> did you mean `retriesBeforeMonFailover` ?

We only use this when a node appears to be drained, so we should name it to give that hint. What about `retriesBeforeNodeDrainFailover`?